server/routes/create-an-activity/handlers/endDate.test.ts

```diff
@@ -88,7 +88,11 @@ describe('Route Handlers - Create an activity schedule - End date', () => {
 
       await handler.POST(req, res)
 
-      expect(res.redirectOrReturn).toHaveBeenCalledWith('days-and-times')
+      expect(res.redirectOrReturnWithSuccess).toHaveBeenCalledWith(
+        '/schedule/activities/undefined',
+        'Activity updated',
+        "We've updated the end date for undefined",
+      )
```

Comment:
Not that keen on validating on `undefined` here. Can this be updated to use a dummy activity?

Reply:
Updated